project/assets/scripts/controller/details/details.js

```diff
@@ -1,49 +1,52 @@
+            setNavEvent('logout', addEvent.bind(null, 'login', redirectToFunction));
+        };
+        this.destroy = function (redirectToFunction) {
+            removeEvents('new-evaluation', addEvent.bind(null, 'newEvaluation', redirectToFunction));
+            removeEvents('evaluations', addEvent.bind(null, 'newEvaluation', redirectToFunction));
+            removeEvents('logout', addEvent.bind(null, 'login', redirectToFunction));
+        };
+        this.populate = function (currentId) {
+            const currentCandidateData = getCurrentCandidateData(currentId);
+            document.getElementById('candidate').value = currentCandidateData.inputData.candidate;
+            document.getElementById('interviewer').value = currentCandidateData.inputData.interviewer;
+            document.getElementById('date').value = currentCandidateData.inputData.date;
 
-        document.getElementById(currentCandidateData[0].technicalLevel).checked = true;
-        document.getElementsByName('level').forEach((el) => {
-            const elem = el;
-            if (!elem.checked) {
-                elem.disabled = true;
-            }
-        });
+            document.getElementById(currentCandidateData.technicalLevel).checked = true;
+            document.getElementsByName('level').forEach((el) => {
+                const elem = el;
+                if (!elem.checked) {
+                    elem.disabled = true;
+                }
+            });
 
-        currentCandidateData[0].textarea.forEach((el) => {
-            const key = Object.keys(el)[0];
-            disableElement(key, el);
-        });
+            currentCandidateData.textarea.forEach((el) => {
+                const key = Object.keys(el)[0];
+                disableElement(key, el);
+            });
 
-        currentCandidateData[0].fieldset.forEach((el) => {
-            const key = Object.keys(el);
-            const insideKeys = Object.keys(el[key]);
-            insideKeys.forEach((e) => {
-                disableElement(e, el[key]);
+            currentCandidateData.fieldset.forEach((el) => {
+                const key = Object.keys(el);
+                const insideKeys = Object.keys(el[key]);
+                insideKeys.forEach((e) => {
+                    disableElement(e, el[key]);
+                });
             });
-        });
-    };
-}
+        };
+    }
 ;
 
-const getCurrentCandidateData = function (currentName) {
-    const evaluationsArray = localStorageGetter('evaluationsArray');
-    const finalArr = evaluationsArray.filter(el =>
-        constructId(el.inputData.candidate) === currentName);
-    return finalArr;
-};
+    const getCurrentCandidateData = function (currentId) {
+        const evaluationsArray = localStorageGetter('evaluationsArray');
+        return evaluationsArray[currentId];
+    };
+
+    feedbackApp.details.events = {
+        setup: new DetailsEvents(),
```

Comment:
Stray comma at end of line (aka EOL).

Reply:
That's also because of ESLint.

Reply:
Gah... I have a hunch something's off with that ESLint config, then...